ingress-operator/pkg/controller/core.go

```diff
@@ -267,11 +269,18 @@ func MakeAnnotations(fni *faasv1.InferenceIngress, host string) map[string]strin
 					"/" + fni.Spec.Function + "/$1"
 				annotations["nginx.ingress.kubernetes.io/use-regex"] = "true"
 			default:
-				annotations["nginx.ingress.kubernetes.io/rewrite-target"] = "/inference/" + fni.Name + "." + inferenceNamespace + "/$1"
-				annotations["nginx.ingress.kubernetes.io/ssl-redirect"] = "false"
-				annotations["nginx.ingress.kubernetes.io/use-regex"] = "true"
+				// for inference created by modelz apiserver
```

Comment:
Is this designed for BYOC created inference when `INGRESS_OPERATOR_SERVICE_HOST` might be not be apiserver?

Reply:
If you set `INGRESS_OPERATOR_SERVICE_HOST` be `apiserver`, you can not use `mdz` create inference in local environment.